src/Controllers/fetchProblems.ts

```diff
@@ -0,0 +1,39 @@
+import { Response } from 'express';
+import { ProblemSetQuestionListData } from '../types';
+
+const fetchProblems = async (
+  options: { limit: number; tags: string },
+  res: Response,
+  formatData: (data: ProblemSetQuestionListData) => {},
+  query: string
+) => {
+  try {
+    const response = await fetch('https://leetcode.com/graphql', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        Referer: 'https://leetcode.com',
+      },
+      body: JSON.stringify({
+        query: query,
+        variables: {
+          categorySlug: '',
+          skip: 0,
+          limit: options.limit || 20, //by default get 20 question
```

Comment:
before this was ```options.limit ? options.limit : 20```, I believe this is the same operation here

Reply:
Yeah! Same